exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Table.kt

```diff
@@ -660,6 +660,13 @@ open class Table(name: String = "") : ColumnSet(), DdlAware {
         defaultValueFun = { defaultValue }
     }
 
+    /** Sets the default value for this nullable column in the database side. */
+    @JvmName("defaultNullableValue")
```

Comment:
Is it really needed to add another `default` function? Will it work if just change `<T : Any>` to `<T>` on the existing one?